mbed_flasher/flashers/FlasherPyOCD.py

```diff
@@ -36,6 +36,10 @@ class PyOCDMap(object):
     Provide list of boards and assisting methods to support PyOCD usage.
     """
     SUPPORTED_PLATFORMS = {
+        "K64F": {
+            "platform": "k64f",
```

Comment:
Does it matter if this is capitalized or not? We tend to capitalize the letters in K64F so it would be good if it was the same here.

Reply:
It does matter. Key has to match mbedls platform and platform value has to match pyocd target.